<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>704</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Printalyzer Densitometer Desktop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabMeasurement">
       <attribute name="title">
        <string>Measurement</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,0,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QGroupBox" name="measResultGroupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>220</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Latest Measurement</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <property name="topMargin">
              <number>6</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="readingTypeNameLabel">
                 <property name="font">
                  <font>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>No Measurement</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_7">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="zeroIndicatorLabel">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="text">
                  <string notr="true"/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="../assets/densitometer.qrc">:/images/zero-set-indicator.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="readingTypeLogoLabel">
                 <property name="text">
                  <string notr="true"/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="../assets/densitometer.qrc">:/images/reflection-icon.png</pixmap>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="readingValueLineEdit">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>18</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string notr="true">0.00D</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QGroupBox" name="measCommandGroupBox">
              <property name="title">
               <string>Commands</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <item>
                <widget class="QPushButton" name="copyTablePushButton">
                 <property name="text">
                  <string>Copy Table</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_12">
                 <item>
                  <widget class="QRadioButton" name="vertCopyRadioButton">
                   <property name="text">
                    <string>Vertical</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">copyDirButtonGroup</string>
                   </attribute>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="horizCopyRadioButton">
                   <property name="text">
                    <string>Horizontal</string>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">copyDirButtonGroup</string>
                   </attribute>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="Line" name="line">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="clearTablePushButton">
                 <property name="text">
                  <string>Clear Table</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_9">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,1">
          <item>
           <spacer name="verticalSpacer_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="autoAddPushButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../assets/densitometer.qrc">
              <normaloff>:/images/media-record.png</normaloff>:/images/media-record.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addReadingPushButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../assets/densitometer.qrc">
              <normaloff>:/images/go-next.png</normaloff>:/images/go-next.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="measTableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDiagnostics">
       <attribute name="title">
        <string>Diagnostics</string>
       </attribute>
       <layout class="QGridLayout" name="tabDiagnosticsLayout">
        <item row="0" column="0">
         <widget class="QWidget" name="tabDiagnosticsWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCalibration">
       <attribute name="title">
        <string>Calibration</string>
       </attribute>
       <layout class="QGridLayout" name="tabCalibrationLayout">
        <item row="0" column="0">
         <widget class="QWidget" name="tabCalibrationWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>592</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="separator"/>
    <addaction name="actionImportSettings"/>
    <addaction name="actionExportSettings"/>
    <addaction name="separator"/>
    <addaction name="actionLogger"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="../assets/densitometer.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;onnect</string>
   </property>
   <property name="toolTip">
    <string>Connect to densitometer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="../assets/densitometer.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from densitometer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="../assets/densitometer.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure application</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../assets/densitometer.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLogger">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../assets/densitometer.qrc">
     <normaloff>:/images/logger.png</normaloff>:/images/logger.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Log Window</string>
   </property>
   <property name="toolTip">
    <string>Show log window</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionExportSettings">
   <property name="text">
    <string>Export Device Settings...</string>
   </property>
   <property name="toolTip">
    <string>Export settings to file</string>
   </property>
  </action>
  <action name="actionImportSettings">
   <property name="text">
    <string>Import Device Settings...</string>
   </property>
   <property name="toolTip">
    <string>Import settings from file</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="edit-cut" resource="../assets/densitometer.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy" resource="../assets/densitometer.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste" resource="../assets/densitometer.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="edit-delete"/>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>readingValueLineEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../assets/densitometer.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="copyDirButtonGroup"/>
 </buttongroups>
</ui>
